### DISCUSSION TITLE
Does Anti-Capitalism affect Tumblr ad revenue?

### DISCUSSION THREAD

**Discussion by kerchanimeprotagonist** [AprilSylph/XKit-Rewritten#498] [Help Desk] [answered]
Hello! 
I’ve been using Anti-Capitalism for a while now, and it works great! I have a question, though. 
The features wiki states that it only _hides_ ads and doesn’t block them. Does this mean that Tumblr still gets ad revenue when I use the site with anti-capitalism enabled as long as my ad-blocker is disabled?
Thank you!

**Comment by AprilSylph** [comment, marked as answer]
As I understand it, the static ads which Tumblr displays are pay-per-click. If you never interacted with ads anyway, I don't think there's a difference.

Ad revenue is a drop in the water anyway. Tumblr costs more than [USD$50 million to run per year](https://photomatt.tumblr.com/post/671876816407052288/) and ads on web are low-value in the first place compared to on mobile apps, where advertisers can have more control. If you're worried about Tumblr's revenue from blocking or hiding ads, I recommend you subscribe to [Tumblr Ad-Free Browsing](https://www.tumblr.com/settings/ad-free-browsing) instead.

**Reply by kerchanimeprotagonist** [reply]
Oh, I see! I was under the impression that it was pay-per-view, but that makes a lot more sense. Thank you for explaining!
